app/toolbox-server/Toolbox/Render.hs

```diff
@@ -70,16 +71,17 @@ renderMainPanel ::
   UIState ->
   ServerState ->
   Widget HTML (Maybe Text)
-renderMainPanel (UIState chan mexpandedModu) (ServerState _ m s) =
+renderMainPanel (UIState chan mexpandedModu) ss =
   case chan of
-    CheckImports -> renderInbox (CheckImports, mexpandedModu) m
+    CheckImports -> renderInbox (CheckImports, mexpandedModu) (serverInbox ss)
     Timing ->
       div
         []
-        [ pre [] [text $ T.pack $ show s]
-        , renderInbox (Timing, mexpandedModu) m
+        [ pre [] [text $ T.pack $ show (serverSessionInfo ss)]
+        , div [] [renderTiming ss]
+        -- , renderTiming ss -- renderInbox (Timing, mexpandedModu) (serverInbox ss)
```

Comment:
Is this leftover?

Reply:
oops. missed this remained in this PR. the following PR #13 removed this line. Thanks for the catch.